agent/taskresource/envFiles/envfile.go

```diff
@@ -0,0 +1,198 @@
+package envFiles
+
+import (
+	"sync"
+	"time"
+
+	apicontainer "github.com/aws/amazon-ecs-agent/agent/api/container"
```

Comment:
can you run gofmt on all the new files? the import sequence looks not gofmt'd.

Reply:
oops will run them